### DISCUSSION TITLE
Graphql with infinite pagination

### DISCUSSION THREAD

**Discussion by galkatz373** [TanStack/query#1364] [Q&A]
I recently discovered this amazing library, I would like to migrate from apollo. But I can't figure out to handle infinite scrolling. I'm using the [graphql-code-generator](https://graphql-code-generator.com/) and it doesnn't seem to have the useInfiniteQuery. 

Thanks in advance.

**Comment by tannerlinsley** [comment]
You’ll have to file an issue there for it to be added. We were not the devs on that generator.

**Reply by galkatz373** [reply]
Will do. Thanks for the fast response.
